azavea/echo-locator
# Replace old logo with new one

## Overview

The log in page and header were attempting to display the old logo. This PR adds the new logo and deletes the old one. 


### Demo
![Screen Shot 2021-11-02 at 3 34 00 PM](https://user-images.githubusercontent.com/77936689/139940250-a958c17f-df28-426a-8c4b-b88c352ed383.png)


![Screen Shot 2021-11-02 at 3 33 51 PM](https://user-images.githubusercontent.com/77936689/139940260-a276624a-e359-4d20-9f34-b81aa18c0d16.png)


### Notes

Prior to this PR, the old logo source path was `'taui/assets/BHAlogo.png'`. I changed the path to direct to where the old logo actually was in `'taui/src/img/'` as the repo ignores `'assets'` and I therefore didn't understand why that would be a good place to store images. I thought maybe `assets` related to deployment and therefore my not being able to see the old logo at the given path could have been related to the fact that I wasn't connected to AWS, so I set up the AWS profile but am still not seeing the original logo. So I went with changing the path, but please let me know if I'm just missing something about Taui, AWS, or the purpose of `assets`.

This PR does not do any styling around the logos, just changes out the old for the new. It also just re-uses the names given to the new logo files rather than naming them to something less wordy. In both cases, I wasn't quite sure what the ideal was so just went with what was already there.


## Testing Instructions

 * Run `./scripts/server`
 * Navigate to the log in page; you should see the new logo at the top of the screen
 * Log in; you should see the new logo in the top left corner


Resolves #353
